testapi.pm

```diff
@@ -1473,6 +1479,23 @@ sub type_password {
     type_string $string, secret => 1, max_interval => ($args{max_interval} // 100), %args;
 }
 
+=head2 enter_cmd
+
+  enter_cmd($string [, max_interval => <num> ] [, wait_screen_changes => <num> ] [, wait_still_screen => <num> ] [, secret => 1 ]
+  [, timeout => <num>] [, similarity_level => <num>] );
+
+A convenience wrapper around C<type_string>, that adds a linefeed to execute a
+command within a command line prompt.
+
+You can pass the same optional parameters as for C<type_string> function.
```

Comment:
Not really important but I'd write it as:
```suggestion
Supports the same optional parameters as the C<type_string> function.
```

It would make sense to remove the parameter list here (to avoid them from becoming out-of-sync).

Reply:
You mean remove the parameter list from the summary string? I know it's tricky to maintain but I would say we can not do better with the very verbose and manual way of handling POD. I don't want to change the existing testapi.pm all over to implement that suggestion consistently